waku.md

```diff
@@ -133,16 +133,16 @@ messages = 1*waku-envelope
 p2p-request = waku-envelope
 p2p-message = 1*waku-envelope
 
-; packet-format needs to be paired with its corresponding 
-; packet-format
+; packet-format needs to be paired with its 
+corresponding packet-format
 packet-format = "[" packet-code packet-format "]"
 
 required-packet = 0 status / 
                   1 messages /
 		  2 pow-requirement /
 		  3 bloom-filter
 		  
-optional-packet = 126 p2p-request / 127 p2p-message
+optional-packet = 126 p2p-request / 127 p2p-message / 14 rate-limits
```

Comment:
ie these are the packet codes, which are (see below)

> All primitive types are RLP encoded. Note that, per RLP specification, integers are encoded starting from 0x00.

So should be 20 rate-limits then here as well I think